### DISCUSSION TITLE
Cannot import UnitofIrradiance

### DISCUSSION THREAD

**Discussion by mat117** [cytech/Home-Assistant-wundergroundpws#145] [Q&A] [answered]
Hello, i upgraded to 1.1.0. I changed config.yaml file as described and inserted my sensor list in the "sensors" file that i import in the config. But i'm getting this error trying to reboot HA: 
` cannot import name 'UnitOfIrradiance' from 'homeassistant.const' (/usr/src/homeassistant/homeassistant/const.py)
`
What is the issue? 
thanks.

**Comment by cytech** [comment, marked as answer]
wundergroundpws v1.1.0 requires Home Assistant Version 2023.1 or greater

**Comment by mat117** [comment]
well, thanks a lot, I was convinced to have latest HA version, but i was wrong...

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
